.lagoon/pgvector/init.sql

```diff
@@ -0,0 +1,13 @@
+-- Enable the pgvector extension so we can store vector embeddings
+CREATE EXTENSION IF NOT EXISTS vector;
+
+-- Create the table that Search API will use for storing embeddings
+CREATE TABLE IF NOT EXISTS search_api_vector (
+  id BIGSERIAL PRIMARY KEY,
+  content VARCHAR,
+  drupal_entity_id VARCHAR,
+  drupal_long_id VARCHAR,
+  server_id VARCHAR,
+  index_id VARCHAR,
+  embedding vector(1024)
+);
```

Comment:
@Leksat can you please remove this? I will handle this inside the module code, there is a service with a method to create collections (which also includes the proper table data). It's ok though to leave the init.sql file (even if empty).

Reply:
As discussed in the call (and later it was confirmed by Philipp), we will leave this table to be used with search_api. And for other services we can create new tables.